File: selection.java
class selection {
    public static void main(String[] args) {
        int[] a = { 78, 52, 12, 45 };
        int min, temp = 0;
        for (int i = 0; i < a.length; i++) {
            min = i;
            for (int j = i + 1; j < a.length; j++) {
                if (a[j] < a[min]) {
                    min = j;
                }
            }
            temp = a[i];
            a[i] = a[min];
            a[min] = temp;
        }
        for (int i = 0; i < a.length; i++) {
            System.out.print(a[i] + " ");
        }
    }
}

File: selection.cpp
// #include<iostream>
// using namespace std;
// int main()
// {
//     int arr[]={55,57,23,76,32}, n=5;

//     for(int i=0; i<(n-1); i++) 
//     {
//         for(int j=(i+1); j<n;j++)
//         {
//             if(arr[j]< arr[i])
//             {
//                 int temp= arr[j];
//                 arr[j]= arr[i];
//                 arr[i]= temp;
//             }
//         }
//     }
//     for(int i=0; i<5;i++)
//     {
//         cout<<arr[i]<<" ";
//     }
// }
#include<iostream>
using namespace std;
int main()
{
    int arr[]={12,45,86,75,19,25,96,112},n=8;
    for(int i=0; i<n-1;i++)
    {
        for(int j=(i+1); j<n; j++)
        {
            if(arr[j]<arr[i])
            {
                int temp= arr[j];
                arr[j] = arr[i];
                arr[i]= temp;
            }
        }
    }
    for(int i=0; i<n; i++)
    {
        cout<<arr[i]<<" ";
    }
}